src/website/webview/views.py

```diff
@@ -518,103 +518,89 @@ def get_queryset(self) -> Any:
         return queryset
 
     def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-        filter_for_suggestions = (
-            self.status_filter != CVEDerivationClusterProposal.Status.REJECTED
-        )
-        title, current_page, new_status, suggestion, cached_suggestion = (
-            update_suggestion(request, filter=filter_for_suggestions)
+        # We want to provide graceful fallback for important workflows, when users have JavaScript disabled
+        js_enabled: bool = "no-js" not in request.POST
```

Comment:
The problem here is that you can't add form parameters specifically for the htmx request, you can only remove them. This is why there is this awkward no-js flag that is filtered out when using htmx.

Reply:
Which is why it would make sense to have a separate endpoint. Doing something like

```
<form method="post" action="" hx-put="/suggestion/{{id}}/state">
    <button type="submit" name="status" value="accepted">Select</button>
    <button type="submit" name="status" value="rejected">Dismiss</button>
</form>
```

would circumvent all that noise entirely. htmx is absolutely beautiful.